### ISSUE TITLE
GUI: support for agglutinative languages 

### ISSUE THREAD

**Issue by heatherleaf** [MUSTE-Project/MULLE#49]
GF has support for agglutination using a special token `BIND`

So, a linearisation like this:

    en blomster BIND flicka slår datorn

Should be rendered thus:

    en blomsterflicka slår datorn

**Comment by fredefox** [comment]
Yeah, I was thinking about this actually. The usage of `words :: Text -> [Text]` is *not* the correct way to break up a string into "words".

**Comment by heatherleaf** [comment]
In 42d88b86b30afb2687707107edc6822fa40087af I added a simple test grammar for agglutination. It uses the GF BIND symbol ("&+") which is the preferred way of glueing words together:
```
$ gf Exemplum.pgf

Exemplum> p -lang=ExemplumEng "the good girl reads" | l -lang=ExemplumTest
the good &+ girl read

Exemplum> p -lang=ExemplumEng "the good girl reads" | l -bind -lang=ExemplumTest
the goodgirl read
```

The grammar works fine in the GUI, but it prints "the good &+ girl read". We want the "&+" to shrink the space to nothing.

**Comment by heatherleaf** [comment]
Simple solution for GUI: make the `<span>` for a "&+" word invisible (add css `display:none` to those spans)

**Comment by fredefox** [comment]
Fixed in 21485ee
